third_party/parser/codegen/generate_diagnostics.cc

```diff
@@ -58,6 +58,7 @@ tuple<string, string> MESSAGES[] = {
     {"InvalidReturn", "invalid return in class/module body"},
     {"CSendInLHSOfMAsgn", "&. inside multiple assignment destination"},
     {"CantAssignToNumparam", "cannot assign to numbered parameter {}"},
+    {"OrdinaryParamDefined", "ordinary parameter is defined"},
```

Comment:
Is this the error that whitequark gives? this is a pretty bad error message in my mind. i'd be ok to change the error message to something like "can't use numbered params when ordinary params were also defined" but up to you.

Reply:
> Is this the error that whitequark gives? 

I think to remember so.

I changed it to "can't use numbered params when ordinary params were also defined" 👍

Reply:
yeah up to you.